forge-gui/res/cardsfolder/p/prismari_apprentice.txt

```diff
@@ -4,7 +4,7 @@ Types:Creature Human Shaman
 PT:2/2
 T:Mode$ SpellCastOrCopy | ValidCard$ Instant,Sorcery | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ Magecraft — Whenever you cast or copy an instant or sorcery spell, CARDNAME can't be blocked this turn. If that spell has mana value 5 or greater, put a +1/+1 counter on CARDNAME.
 SVar:TrigPump:DB$ Pump | Defined$ Self | KW$ HIDDEN Unblockable | SubAbility$ DBPutCounter
-SVar:DBPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ 1 | ConditionCheckSVar$ TriggerCount$CastSACMC | ConditionSVarCompare$ GE5
+SVar:DBPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ 1 | ConditionCheckSVar$ TriggeredStackInstance$CardManaCostLKI | ConditionSVarCompare$ GE5
```

Comment:
wouldn't that cause problems with Param Split?

Reply:
No, it's actually safe:
`parseToMap` only splits first $ between each |

Reply:
but wasn't Zur's Weirding crashing because of `Description$ Description$` ? https://github.com/Card-Forge/forge/commit/d17ab3dce1a34e07e7f45cf18e7c709db67b87b2

Reply:
Ugh, I didn't know for some reason `StaticAbility` still uses its own `parseParams` instead of core `FileSection` class compared to all other Factories

I think I will fix that so it can use cache for performance as seen here: https://github.com/Card-Forge/forge/commit/dd4df9baaa70ef5a2eea44bfaeea5393b68cb4a8